articles/team-fit.md

```diff
@@ -0,0 +1,35 @@
+
+Software engineering is highly collaborative process. From its very beginning, in discovery phase, until its retirement we’re in constant contact with various stakeholders. Furthermore, as we take on more responsibility, collaboration become more important.
+
+Software is everywhere. Professional services takes us to diverse industry sectors. Creating solution that will meet end-user expectations, require from us to really understand the business, regardless of the role.
+
+To work, contribute and progress in such system, we need to be equipped with competencies beyond technical. For that we'll look in domain of emotional and social intelligence. Former provides skills to execute the task, while the latter enable us to take it through the process efficiently.
+
+So, what is that, exactly, we are looking for? This is a question I brought to colleagues. After several individual and group discussions, we came to the consensus on the list of eight compatencies, of which all are perceived as crucial in certain context:
+
+- **Approachable:** Prerequisite for collaboration to happen is that you feel comfortable to approach your colleague no matter if you bring bad or good news. 
+- **Accountable:** _They may rely on you_. Once you came to agreement with your team mates, and the work is shared, they should be able to trust you that you’ll complete your tasks in agreed timeframe and quality.
+- **Integrity:** May your actions be _the one_ with your values, principles and beliefs, in every context. People of integrity always do the right thing. They are honest and feel comfortable to say when don’t know. They are open for different opinion and always welcome feedback.
+- **Empathy:** Only if you’re honestly concerned and understand others’ feelings and perspective you’ll be able to establish a mutual bond. That bond is fundamental for establishing a feeling of psychological safety in the team – base of every high-performance team.
+- **Adaptability:** We live in the world of diversities. People we work with come from different cultures, opinions and professional/education backgrounds. Projects we work on are from different industries ran by various technologies. As an integral part of these processes, we absorb, analyze and assimilate a huge amount of information and emotions. We adapt.
+- **Pro-active:** In the time of innovations, products do not only respond on impulses from market in real-time, but also try to predict them and act ahead of time. Engineers follow that rhythm, and assisted by proven practices, methodologies and right technology choices bring this to reality.
+- **Courage:** It takes courage to challenge the status quo and going out of comfort zone. Those are first two steps on the way forward – road between opportunity and success. 
+- **Endurance:** _Good things take time_. Just try to remember how many things, that made an impact or you’re proud of, took short time to happen.
```

Comment:
Just try to remember how many things, that made an impact or you’re proud of, took short time to happen.
->
Just try to remember how many things, that you’re proud of, took short time to happen.
-> or even
How many things, that you’re proud of, took short time to happen?